wiki/Performance_points/Performance_points_commitee/en.md

```diff
@@ -0,0 +1,21 @@
+---
+stub: true
+---
+
+# Performance Points Committee
+
+The **Performance Points Committee** is a panel consisting of well-versed members of the development community whose main goal is to ensure that changes in the [performance points](/wiki/Beatmap) calculation algorithm could be facilitated, reviewed, and made effective in a fast and steady flow.
+
+The panel was first announced to the public through the [November 2021 Performance Points & Star Rating Updates newspost](https://osu.ppy.sh/home/news/2021-11-09-performance-points-star-rating-updates).
```

Comment:
Should probably also mention that it was called into existence in response to the community survey from this year.

Reply:
added